### ISSUE TITLE
No estimates for reading (time, current file)

### ISSUE THREAD

**Issue by desperex** [insidegui/F3X#2]
Thanks for the app first of all!
It would be really nice to have estimated time left or the name of file being read to be displayed!
Stuck in reading process for a few hours and don't know when it's gonna finish

**Comment by insidegui** [comment]
I was waiting on an [update to F3](https://github.com/AltraMayor/f3/issues/21) to be able to report the progress. They just updated It with my request so I'll be implementing this as soon as possible.

**Comment by desperex** [comment]
Thanks!

**Comment by asheenlevrai** [comment]
+1

**Comment by asheenlevrai** [comment]
This was updated in f3 like 3 months ago, right?
